scripts/release/merge-release-notes.js

```diff
@@ -2,23 +2,30 @@
 
 const fs = require("fs");
 
+async function listReleases(github, owner, repo) {
+    const response = await github.rest.repos.listReleases({
+        owner,
+        repo,
+        per_page: 100,
+    });
+    // Filters out draft releases
+    return response.data.filter((release) => !release.draft);
+}
+
 // Dependency can be the name of an entry in package.json, in which case the owner, repo & version will be looked up in its own package.json
-// Or it can be a string in the form owner/repo@tag
+// Or it can be a string in the form owner/repo@tag - in this case the tag is used exactly to find the release
+// Or it can be a string in the form owner/repo~tag - in this case the latest tag in the same major.minor.patch set is used to find the release
 // Or it can be a tuple of dependency, from version, to version, in which case a list of releases in that range (to inclusive) will be returned
 async function getReleases(github, dependency) {
     if (Array.isArray(dependency)) {
         const [dep, fromVersion, toVersion] = dependency;
         const upstreamPackageJson = getDependencyPackageJson(dep);
         const [owner, repo] = upstreamPackageJson.repository.url.split("/").slice(-2);
 
-        const response = await github.rest.repos.listReleases({
-            owner,
-            repo,
-            per_page: 100,
-        });
+        const unfilteredReleases = await listReleases(github, owner, repo);
         // Only include non-draft & non-prerelease releases, unless the to-release is a pre-release, include that one
-        const releases = response.data.filter(
-            (release) => !release.draft && (!release.prerelease || release.tag_name === `v${toVersion}`),
+        const releases = unfilteredReleases.filter(
```

Comment:
Can we do this behavior in the `listReleases` too ? Since we are already filtering the non-draft PR in it